lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -356,6 +352,29 @@ class CategoryRenderer {
     return tmpl;
   }
 
+  /**
+   * Define the score arc of the gauge
+   * Credit to xgad for the original technique: https://codepen.io/xgad/post/svg-radial-progress-meters
+   * @param {SVGCircleElement} elem
+   * @param {number} percent
+   */
+  _setGaugeArc(elem, percent) {
```

Comment:
since it's operating on the passed-in score gauge, and not a `this.gauge` or something, make this a static method?

Reply:
> since it's operating on the passed-in score gauge, and not a `this.gauge` or something, make this a static method?

OTOH, there's no other static methods in this file, and lots of them are acting more or less statically too